app/views/components/gds/gds_main_template.scala.html

```diff
@@ -40,7 +40,7 @@
   useCustomContentWidth: Boolean = false
 )(contentBlock: Html)(implicit request: Request[_], messages: Messages)
 
-@useTimeoutDialog = @{ request.isInstanceOf[AuthenticatedRequest[_]]}
+@useTimeoutDialog = @{ request.isInstanceOf[AuthenticatedRequest[_]] || request.isInstanceOf[JourneyRequest[_]] }
```

Comment:
I can see why you are making this change.  I'm not sure how this fix is not required in customs-declare-frontend

Reply:
Journey Request in customs-declare-exports-frontend extends AuthenticatedRequest

https://github.com/hmrc/customs-declare-exports-frontend/blob/master/app/models/requests/JourneyRequest.scala

Reply:
In internal service journeyRequest just contains the authenticatedRequest inside